src/auditor/checks.py

```diff
@@ -298,11 +298,36 @@ def title_check(self):
         #: Always include the old title for readability
         title_data = {'title_fix': 'N', 'title_old': self.item.title, 'title_new': ''}
 
-        if self.new_title and self.new_title != self.item.title:
-            title_data = {'title_fix': 'Y', 'title_old': self.item.title, 'title_new': self.new_title}
+        new_title = None
 
+        #: If we have a new title from metatable, add and check for deprecated note
+        if self.title_from_metatable and self.title_from_metatable != self.item.title:
```

Comment:
**question**: is the null check necessary if there is an inequality check?
```suggestion
        if self.title_from_metatable != self.item.title:
```

Reply:
yes, because `self.title_from_metatable` is initialized to `None`, and then if the conditional is true, this value is assigned to `new_title` on the next line. Thus, `None` would not equal whatever the title was, and the new title would then be set to `None`.

Reply:
Ideally, we won't have a record in the metatable without a title (and thus `self.title_from_metatable` would always be a sane value), but if we did, this keeps it from changing the title to `None`